### PR TITLE
Fix regression from the addition of room_uri in the helper

### PR DESCRIPTION
**Pull request by thiblahute**

### CODE REVIEW

File: helper/cli.rs [hunk 1]
```diff
@@ -51,6 +51,7 @@ pub async fn parse_cli() -> Option<Settings> {
 
     if !credentials.is_complete()
     {
+        eprintln!("===> Incomplete credits!");
```
Comment (philn) [inline helper/cli.rs:54]: debug prints in this file are not needed i guess :)

Comment (thiblahute) [reply helper/cli.rs:54]: Well, other paths did have debug prints, and those were useful so I think it is a good idea to add them.

Comment (philn) [reply helper/cli.rs:54]: Ah yes you added those in another PR. Whatever :) Then please at least reword this to `credentials`.

Comment (thiblahute) [reply helper/cli.rs]: Fixed.

**Review by philn — APPROVED** [review]
